tests/utils/test_skiperror.py

```diff
@@ -0,0 +1,29 @@
+import pandas as pd
+from janitor.utils import skiperror
+import numpy as np
+
+import pytest
+
+
+@pytest.mark.functions
+def test_skiperror():
+    df = pd.DataFrame({"x": [1, 2, 3, "a"], "y": [1, 2, 3, "b"]})
+    func = lambda s: s + 1
+
+    # Verify that applying function causes error
```

Comment:
Lines 14-18: These should be caught using a `with pytest.raises` block. You can find an example [here](https://stackoverflow.com/a/56569533/1274908).